content/blog/2020-05-rules-0-0-1/index.Rmarkdown

```diff
@@ -0,0 +1,122 @@
+---
+title: rules 0.0.1
+slug: rules-0-0-1
+description: >
+    The rules package is on CRAN and enables rule-based models for tidymodels.
+date: 2020-05-18
+author: Max Kuhn
+photo:
+  url: https://unsplash.com/photos/lu15z1m_KfM
+  author: Lili Popper
+categories:
+  - package
+---  
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(
+  collapse = TRUE, comment = "#>", 
+  fig.width = 7, 
+  fig.align = 'center',
+  fig.asp = 0.618, # 1 / phi
+  out.width = "700px"
+)
+library(doMC)
+registerDoMC(cores = parallel::detectCores())
+```
+
+We are happy to announce the release of the [rules package](https://rules.tidymodels.org) on [CRAN](https://cran.r-project.org/package=rules). rules is another "parsnip-adjacent" package that enables a specific class of models within the tidymodels infrastructure.  rules currently contains three models:
+
+ * `C5_rules()`: classification rule sets based on the C5.0 model.
+ 
+ * `cubist_rules()`: regression rules using Cubist.
+ 
+ * `rule_fit()`: classification or regression rules using the RuleFit model. 
+ 
```

Comment:
Add the typical chunk here about "To install this package run…" Here's how Jenny does it:
https://github.com/tidyverse/tidyverse.org/blob/8cc480b9cf16b3076bfbdf19e8af1aad30521d4f/content/blog/2020-05-googlesheets4-0-2-0/index.Rmarkdown#L40-L50